packages/fxa-settings/src/components/Settings/UnitRowTwoStepAuth/index.tsx

```diff
@@ -39,95 +38,6 @@ export const UnitRowTwoStepAuth = () => {
     useBooleanState();
   const ftlMsgResolver = useFtlMsgResolver();
 
-  const disableTwoStepAuth = useCallback(async () => {
```

Comment:
Noticed a bug where glean view metrics for this modal were emitted multiple times - moved the subcomponent out of the main component to resolve. Also added a proper session verification guard